src/rust/lib/flexer/src/parser.rs

```diff
@@ -0,0 +1,6 @@
+use crate::automata::state::Symbol;
+
+
+
+pub const EOF_CODE:Symbol = Symbol{val:-1};
+pub const ETX_CODE:Symbol = Symbol{val:-2};
```

Comment:
this is such a magic that it deserves big docs.
+ no new line
+ no section
+ no docs in file